webpack.config.js

```diff
@@ -263,6 +263,8 @@ const webpackConfig = {
 		modules: [ path.join( __dirname, 'client' ), 'node_modules' ],
 		alias: Object.assign(
 			{
+				react: path.dirname( require.resolve( 'react' ) ),
+				'react-dom': path.dirname( require.resolve( 'react-dom' ) ),
```

Comment:
Had to add a react alias because duplicated react was throwing errors. It believes hooks are being used outside of a function component.

Reply:
This was fixed by rolling back the `format-library` package.